test.env.example

```diff
@@ -1,5 +1,19 @@
-REDSHIFT_TEST_HOST=
-REDSHIFT_TEST_USER=
-REDSHIFT_TEST_PASS=
-REDSHIFT_TEST_PORT=
-REDSHIFT_TEST_DBNAME=
+# Note: Make sure you have a Redshift account that is set up so these fields are easy to complete. 
+If you don't have an account set up yet, then take note of these required fields that way. When you're getting set up, 
+you can use them later to build your Redshift project.
+
+### Test Environment field definitions 
+# These will all be gathered from account information or created by you.
+
+REDSHIFT_TEST_HOST: Endpoint for Redshift connection
+REDSHIFT_TEST_USER: Username on your account
+REDSHIFT_TEST_PASS: Password for Redshift account
+REDSHIFT_TEST_PORT: Local port to connect on.
+REDSHIFT_TEST_DBNAME: Name of Redshift database in your account to test against.
+
+# Copy the follwing to a test.env and replace example values with your information.
+REDSHIFT_TEST_HOST=host_name_for_account
+REDSHIFT_TEST_USER=user_name_for_account
+REDSHIFT_TEST_PASS=your_password
+REDSHIFT_TEST_PORT=5439
+REDSHIFT_TEST_DBNAME=name_of_database
```

Comment:
```suggestion
```
There are already instructions in `CONTRIBUTING.md` to copy the file and edit values. No need for these instructions 👍

Reply:
Done!